libs/core/langchain_core/messages/tool.py

```diff
@@ -82,15 +83,21 @@ def get_lc_namespace(cls) -> List[str]:
         Default is ["langchain", "schema", "messages"]."""
         return ["langchain", "schema", "messages"]
 
+    @model_validator(mode="before")
+    @classmethod
+    def coerce_args(cls, values: dict) -> dict:
+        content = values["content"]
```

Comment:
Since this is the response from an invocation and is expected to go to a chat model I'm OK with a more generic str() invocation regardless of type

try:
   str(content)
except ValueError:
   some useful error